imgoing-frontend/src/components/common/Input.tsx

```diff
@@ -29,18 +30,34 @@ interface OwnProps {
 
 const StyledInput = styled(TextInput)<OwnProps & InputProps>`
   width: 100%;
-  height: 50px;
   background: ${({ theme }) => theme.colors.white};
   border: 2px solid ${({ theme, isFocus }) => (isFocus ? theme.colors.blue : theme.colors.black)};
   border-radius: 4px;
   font-size: 16px;
   padding: 13px 16px 14px 16px;
+  ${(props) => {
+    if (props.long)
+      return css`
+        height: 150px;
+      `;
+    return css`
+      height: 50px;
+    `;
+  }}
 `;
+
 const InputWrapper = styled.View``;
 
+const WordCountView = styled.View`
+  align-items: flex-end;
+  padding: 5px;
+`;
+
 const Input = (props: InputProps) => {
-  const { style, onChange, name, ...restProps } = props;
+  const { style, onChange, name, long, ...restProps } = props;
   const [isFocus, setFocus] = useState<boolean>(false);
+  const [wordCount, setWordCount] = useState<number>(0);
```

Comment:
```suggestion
  const { style, onChange, name, long, value, ...restProps } = props;
  const [isFocus, setFocus] = useState<boolean>(false);
  const [wordCount, setWordCount] = useState<number>(value.length || 0);
```

일정 편집은 props로 value가 넘어와서 길이를 구할 수 있을 것 같아요!
그리고 StyleInput에 props로 value 따로 넣어주는 거 잊지마세요~~

Reply:
아 간단하게 할수있었네요~~ 최곱니다~